packages/react-native/android/src/main/java/com/bugsnag/reactnative/ConfigSerializer.java

```diff
@@ -0,0 +1,25 @@
+package com.bugsnag.reactnative;
+
+import com.bugsnag.android.Configuration;
+
+import com.facebook.react.bridge.WritableMap;
+import com.facebook.react.bridge.WritableNativeMap;
+
+public class ConfigSerializer implements WritableMapSerializer<Configuration> {
+
+    @Override
+    public WritableMap serialize(Configuration config) {
+        WritableMap map = new WritableNativeMap();
+        map.putString("apiKey", config.getApiKey());
+        map.putString("appVersion", config.getAppVersion());
+        map.putString("buildUuid", config.getBuildUUID());
+        map.putString("endpoint", config.getEndpoint());
```

Comment:
Is it possible to write the two endpoints into a nested `endpoints` map:

```java
WritableMap endpoints = new WritableNativeMap();
endpoints.put("notify", config.getEndpoint());
endpoints.put("sessions", config.getSessionEndpoint());
map.putMap("endpoints", endpoints);
```

And if so should we? 🤔 it matches the JS interface that way, but maybe that's not important.

Reply:
It's easy enough to update the format to match JS - and is probably worth the consistency.